gbs-control.ino

```diff
@@ -6917,8 +6941,39 @@ void loadDefaultUserOptions() {
 //  //system_phy_set_powerup_option(3); // 0 = default, use init byte; 3 = full calibr. each boot, extra 200ms
 //  system_phy_set_powerup_option(0);
 //}
-
+void ICACHE_RAM_ATTR isr (){
+  static unsigned long lastInterruptTime = 0;
+  unsigned long inturruptTime = millis();
+
+  if(inturruptTime - lastInterruptTime > 120){
+    if(!digitalRead(pin_data)){
+      encoder_pos++;
+      main_pointer+=15;
+      sub_pointer+=15;
+      pointer_count++;
+      // down = true;
+      // up = false;
+    } else{
+      encoder_pos--;
+      main_pointer-=15;
+      sub_pointer-=15;
+      pointer_count--;
+      // down = false;
+      // up = true;
+    }
+  }
+  lastInterruptTime = inturruptTime;
+}
 void setup() {
+  display.init();
```

Comment:
displayOLED?

Reply:
Ah okay, then a comment will do. A non-existing OLED display device on the I2C bus will simply not respond, and the code move on, right?

Reply:
Yes, if no OLED device is connected on the bus, it will compile fine and operate as normal :)

Reply:
Comments added. Edited to reflect this change